lib/encrypt.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/enigma'
+
+enigma = Enigma.new
+
+handle = File.open(ARGV[0], "r")
+
+message = handle.read
+
+handle.close
+
+encrypted_info = enigma.encrypt(message)
+
+writer = File.open(ARGV[1], "w")
+
+writer.write(encrypted_info[:encryption])
+
+writer.close
+
+puts "Created '#{writer.path}' with the key #{encrypted_info[:key]} and date #{encrypted_info[:date]}"
```

Comment:
The random key and date generated by `enigma.encrypt` are reported here, and should be used as command line arguments when calling `decrypt.rb`